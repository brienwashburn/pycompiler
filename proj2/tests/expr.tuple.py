(a, b, c)
(a, b, c,)
(a + b, c, q)
(a,)
(a)
a,
a, b, c, d
a, b, c, d,
