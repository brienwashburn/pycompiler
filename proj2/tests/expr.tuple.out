(Module
 (Expr (Tuple (Name a) (Name b) (Name c)))
 (Expr (Tuple (Name a) (Name b) (Name c)))
 (Expr (Tuple (BinOp (Name a) Add (Name b)) (Name c) (Name q)))
 (Expr (Tuple (Name a)))
 (Expr (Name a))
 (Expr (Tuple (Name a)))
 (Expr (Tuple (Name a) (Name b) (Name c) (Name d)))
 (Expr (Tuple (Name a) (Name b) (Name c) (Name d))))
